### PR TITLE
Make packager.sh compatible with Jenkins on Ubuntu 14.04

### PR DESCRIPTION
**Pull request by daniviga**
This PR fixes `packger.sh` to run on CI hosted by an Ubuntu 14.04. It doesn't fix issues for creating packages in an Ubuntu 14.04 LXC. The LXC command line tools have changed a bit with LXC >= 1.0

Test for CI on 12.04: https://ci.openquake.org/job/zdevel_oq-risklib/344/

### CODE REVIEW

**Comment by micheles** [comment]
LGTM

File: packager.sh [hunk 1]
```diff
@@ -361,8 +371,8 @@ _lxc_name_and_ip_get()
             lxc_name="$(grep "sudo lxc-console -n $GEM_EPHEM_NAME" /tmp/packager.eph.$$.log | sed "s/.*sudo lxc-console -n \($GEM_EPHEM_NAME\)/\1/g")"
             for e in $(seq 1 40); do
                 sleep 2
-                if grep -q "$lxc_name" /var/lib/misc/dnsmasq.leases ; then
-                    lxc_ip="$(grep " $lxc_name " /var/lib/misc/dnsmasq.leases | cut -d ' ' -f 3)"
+                if grep -q "$lxc_name" /var/lib/misc/dnsmasq*.leases ; then
+                    lxc_ip="$(grep " $lxc_name " /var/lib/misc/dnsmasq*.leases | cut -d ' ' -f 3)"
                     break
```
Comment (nastasi-oq) [inline packager.sh]: Please add ```| tail -n 1``` just to be sure to take the latest item.

**Comment by daniviga** [comment]
I'm re-running tests. Please do not merge.

**Comment by daniviga** [comment]
Green: https://ci.openquake.org/job/zdevel_oq-risklib/345/

**Pull request merged**